src/styles/blocks/calendar.scss

```
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: $cellSize * 7 + $cellGgap * 6 + $calendaPadding * 2;
+  padding: $calendaPadding;
+
+  &__day {
+    position: relative;
+    @include calendar-cell($cellSize);
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: transform, background-color 0.5s;
```

This way it will work smoothly.

```suggestion
    transition: transform 0.5s, background-color 0.5s;
```